### ISSUE TITLE
info_nce_loss

### ISSUE THREAD

**Issue by zbbwss** [zhanxlin/Product1M#6]

**Comment by zbbwss** [comment]
self.loss_fct = CrossEntropyLoss(ignore_index=-1)
self.loss_fct(logits, labels)

**Comment by zbbwss** [comment]
写的很好 膜拜大佬
